src/officehours_api/views.py

```diff
@@ -2,36 +2,41 @@
 import logging
 from datetime import datetime, timezone, timedelta
 from random import randint
-from django.db import connection
+
+from asgiref.sync import async_to_sync
 from django.conf import settings
 from django.contrib.auth.models import User
+from django.core.exceptions import ObjectDoesNotExist
+from django.db import connection
 from django.http import HttpResponse
 from django.shortcuts import get_object_or_404
-from django.core.exceptions import ObjectDoesNotExist
+from django_filters.rest_framework import DjangoFilterBackend
 from drf_spectacular.utils import extend_schema, inline_serializer
 from rest_framework import generics, serializers, status, filters
+from rest_framework.decorators import api_view
+from rest_framework.permissions import IsAuthenticated
 from rest_framework.response import Response
 from rest_framework.reverse import reverse
-from rest_framework.decorators import api_view
 from rest_framework.views import APIView
 from rest_framework_tracking.mixins import LoggingMixin
-from rest_framework.permissions import IsAuthenticated
-from django_filters.rest_framework import DjangoFilterBackend
-from asgiref.sync import async_to_sync, sync_to_async
 
-from officehours_api.notifications import send_one_time_password
-from officehours_api.exceptions import DisabledBackendException, MeetingStartedException, TwilioClientNotInitializedException
+from officehours_api.exceptions import DisabledBackendException, \
+    MeetingStartedException, TwilioClientNotInitializedException
 from officehours_api.models import Attendee, Meeting, Queue
-from officehours_api.serializers import (
-    ShallowUserSerializer, MyUserSerializer, ShallowQueueSerializer, QueueAttendeeSerializer,
-    QueueHostSerializer, MeetingSerializer, AttendeeSerializer, PhoneOTPSerializer
-)
-from officehours_api.permissions import (
-    IsAssignee, IsHostOrReadOnly, IsHostOrAttendee, is_host
-)
+from officehours_api.notifications import send_one_time_password
+from officehours_api.permissions import (IsAssignee, IsHostOrReadOnly,
```

Comment:
Formatting nitpick, but these imports should probably be on separate lines like the import below is. For instance

```
from officehours_api.permissions import (IsAssignee, 
                                         IsHostOrReadOnly,
                                         IsHostOrAttendee, 
                                         is_host)
```

Reply:
Yeah, perhaps.

The `import` formatting was done entirely by the IDE.  I didn't change anything there after running the reformatter.  My guess is that its goal isn't to put them on individual lines, but to put them in lines where the left side is aligned with the open parenthesis and the right side is no longer than the line length limit (80 characters).

Note that in the import you referenced below, line 34 has `MeetingSerializer, AttendeeSerializer,`, 2 symbols on a line, not only 1.  That's because they're short enough to fit together within the line length limit.